src/app/search/search-results/search-results.component.ts

```diff
@@ -52,6 +55,11 @@ export class SearchResultsComponent implements OnInit {
   ngOnInit() {
     this.createTagCloud('tool');
     this.createTagCloud('workflow');
+    this.selectedIndex$ = this.searchQuery.activeToolTab$.pipe(
+      map(activeToolTab => {
```

Comment:
This is stylistic that I don't think we have an official rule on, so feel free to ignore, but I prefer to use curly braces only when you have statements. You could have instead, if I'm not getting confused with Java, something like:

```
map(activeToolTab => {active: activeToolTab})
```

Reply:
Ive tried getting rid of the return statement, but that resulted in the label active being unnecessary and unused, which caused some errors.